pkg/util/sets/uint.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
Less code is better ? 
Also x := 10 will give you an int, so if we ever want to do an exact compare anywhere uint would lead to unnecessary casting